eta-meeting-organizer-frontend/src/app/login/components/login.component.ts

```diff
@@ -0,0 +1,126 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+
+import { Router } from '@angular/router';
+import {
+  AuthService,
+  AuthServiceConfig,
+  SocialUser,
+} from 'angularx-social-login';
+import { GoogleLoginProvider } from 'angularx-social-login';
+import { ConfigurationService } from '~/app/shared/services/configuration.service';
+import { Subscription } from 'rxjs';
+
+const config = new AuthServiceConfig([
+  {
+    id: GoogleLoginProvider.PROVIDER_ID,
+    provider: new GoogleLoginProvider(
+      '160652702041-14ipa76q95t63j3o974o3focpjsr51i7.apps.googleusercontent.com'
+    )
+  }
+]);
+
+export function provideConfig() {
+  return config;
+}
+
+@Component({
+  selector: 'app-login',
+  styles: [
+    `
+      #row {
+        margin: auto auto;
+        padding: auto 0;
+        min-height: calc(100vh - 60px);
+      }
+      #customBtn {
+        display: inline-block;
+        background: rgb(51, 51, 51);
+        color: rgb(243, 245, 237);
+        width: 190px;
+        border-radius: 5px;
+        box-shadow: 1px 1px 1px grey;
+        white-space: nowrap;
+      }
+      #customBtn:hover {
+        cursor: pointer;
+      }
+      span.label {
+        font-family: serif;
+        font-weight: normal;
+      }
+      img {
+        background: transparent 5px 50% no-repeat;
+        display: inline-block;
+        vertical-align: middle;
+        width: 42px;
+        height: 42px;
+      }
+      span.buttonText {
+        display: inline-block;
+        vertical-align: middle;
+        padding-left: 42px;
+        padding-right: 42px;
+        font-size: 14px;
+        font-weight: bold;
+        font-family: "Roboto", sans-serif;
+      }
+      mat-card {
+        background-color: rgb(230, 75, 58);
+      }
+      mat-card-title,
+      mat-card-content {
+        color: rgb(243, 245, 237);
+        display: flex;
+        justify-content: center;
+      }
+    `,
+  ],
```

Comment:
custom css hegyek

Reply:
Megcsináltam a Material Színek felülírását egy cikk alapján, beraktam egy theme.css-et. A többiek is ahol lehet refaktorálják a saját kódjukat majd. Visszavágtam a css-eket is, a mat-card-nál az volt a bajom, hogy alapesetben hiába adtam meg a color="primary/accent/warn"-ot mindig a szürke volt a háttérszín. Annyi, hogy a google button miatt a span, image, btn elemek elég sok css-t hoztak magukkal.